src/Console/GenerateDocuCommand.php

```diff
@@ -17,46 +15,32 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use UnexpectedValueException;
```

Comment:
I suggest not using native php classes in use statements. It can lead to potential confusion when some people tend to name their own classed the same way.

Reply:
ok